### PR TITLE
fix deflate column decode order

### PR DESCRIPTION
**Pull request by orionz**
Simple fix to remove deflate bit from column decode

### CODE REVIEW

File: automerge-backend-wasm/Cargo.toml [hunk 1]
```diff
@@ -35,12 +35,16 @@ serde_bytes = "0.11.5"
 version = "^0.2"
 features = ["serde-serialize"]
 
+[package.metadata.wasm-pack.profile.release]
+wasm-opt = false
+
```
Comment (jeffa5) [inline automerge-backend-wasm/Cargo.toml:40]: Is this related?

**Review by jeffa5 — COMMENTED** [review]
Looks good, just the one seemingly unrelated bit.

File: automerge-backend-wasm/Cargo.toml (hunk 1 above)
Comment (orionz) [reply automerge-backend-wasm/Cargo.toml:40]: Good catch...  I needed that to get the tests to pass on my new M1 MacBook but it should not be in the PR!
